CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ This release brings many new features, such as stream proxy, support MQTT protoc
 and support for ARM platform, and proxy rewrite plugin.
 
 ### Core
+- :sunrise: **[support stand-alone mode](https://github.com/iresty/apisix/blob/master/doc/stand-alone-cn.md)**: using yaml to update configurations of APISIX, more friendly for kubernetes. [464](https://github.com/iresty/apisix/pull/464)
```

Comment:
`more friendly to kubernetes` would be better

Reply:
thx